### ISSUE TITLE
update development-setup.md for Arrangement 6

### ISSUE THREAD

**Issue by ktdreyer** [containerbuildsystem/osbs-client#895]
I'm setting up a local OSBS instance on CentOS 7, and I'm following this document: https://github.com/containerbuildsystem/osbs-client/blob/master/docs/development-setup.md

I was able to do simple container builds with my `osbs` client, but I cannot get my instance to do orchestrated builds.

Would you please update the document with step-by-step instructions for Arrangement 6?

Also, there is a section "Setting up a (local) development environment" at https://osbs.readthedocs.io/en/latest/contributors.html . Should we move this development-setup document there?

**Comment by MartinBasti** [comment]
In future we plan to replace that section with osbs-box which is able to deploy local test env. osbs-box is under reconstruction now, however with:
- https://github.com/containerbuildsystem/atomic-reactor/pull/1285
- https://github.com/containerbuildsystem/osbs-box/pull/99

It should be able to provide working dev env on local okd instance (`oc cluster up`)

**Comment by ktdreyer** [comment]
I can understand why you recommend osbs-box. I've studied what osbs-box does, and I haven't tried it yet on my laptop, but I want to do so.

It looks like osbs-box uses `docker-compose`, and I suspect that is going to deteriorate on Fedora as RH does not invest in that tool very much. I saw there's some work towards making it all run in an OpenShift cluster instead, but as an OpenShift noob I had a hard time understanding exactly what is going on. It looks like there are certain things in osbs-box that are somewhat experimental and it's hard to sort out what the happy path *should* be.

In my case, I want this to be a long-running thing that integrates with a pre-existing Koji environment. If anyone else uses OSBS in production, they will want to have step-by-step instructions too. I think there is a lot of value in explaining each piece step-by-step, because it teaches the concepts to new admins.

1. Setting up a basic OpenShift cluster (`oc cluster up`) and enabling whatever services and access that you will need,
2. Do a really basic "simple build" with `osbs-client` on the CLI,
3. Do a simple build with `osbs-client` and push it to a registry,
4. Enable the settings for the orchestrator and do an orchestrator build,
5. Add in the kojid part and do a sample build with `koji container-build`
